examples/purchase-tester/src/main/res/layout/fragment_configure.xml

```diff
@@ -0,0 +1,121 @@
+                app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/sdk_configuration_label"
+                style="@style/TextAppearance.AppCompat.Headline"
+                android:text="@string/sdk_configuration"
+                android:layout_marginTop="20dp"
+                app:layout_constraintTop_toBottomOf="@id/configure_icon"
+                app:layout_constraintStart_toStartOf="parent"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"/>
+
+        <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/api_key_label"
+                android:text="@string/api_key"
+                android:textSize="15sp"
+                android:layout_marginTop="30dp"
+                app:layout_constraintTop_toBottomOf="@id/sdk_configuration_label"
+                app:layout_constraintStart_toStartOf="parent"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"/>
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/api_key_input"
+                android:background="@color/lightGrey"
+                app:layout_constraintTop_toTopOf="@id/api_key_label"
+                app:layout_constraintBottom_toBottomOf="@id/api_key_label"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintWidth_percent="0.55"
+                android:layout_marginStart="20dp"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"/>
+
+        <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/proxy_url_label"
+                android:text="@string/proxy_url_optional"
+                android:textSize="15sp"
+                android:layout_marginTop="20dp"
+                app:layout_constraintTop_toBottomOf="@id/api_key_label"
+                app:layout_constraintStart_toStartOf="parent"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"/>
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/proxy_url_input"
+                android:background="@color/lightGrey"
+                app:layout_constraintTop_toTopOf="@id/proxy_url_label"
+                app:layout_constraintBottom_toBottomOf="@id/proxy_url_label"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintWidth_percent="0.55"
+                android:layout_marginStart="20dp"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"/>
+
+        <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/store_label"
+                android:text="@string/store"
+                android:textSize="15sp"
```

Comment:
There is a bunch of things here that could be extracted to styles... But I didn't want to get too much into it.